contracts/tokenomics/maker/src/contract.rs

```diff
@@ -910,41 +909,6 @@ pub fn query_pair(deps: Deps, contract_addr: Addr) -> StdResult<[AssetInfo; 2]>
 ///
 /// * **_msg** is an object of type [`MigrateMsg`].
 #[cfg_attr(not(feature = "library"), entry_point)]
-pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
-    let contract_version = get_contract_version(deps.storage)?;
-
-    match contract_version.contract.as_ref() {
-        "astroport-maker" => match contract_version.version.as_ref() {
-            "1.0.0" => {
-                let config_v100 = migration::CONFIGV100.load(deps.storage)?;
-
-                let new_config = Config {
-                    owner: config_v100.owner,
-                    factory_contract: config_v100.factory_contract,
-                    staking_contract: config_v100.staking_contract,
-                    governance_contract: config_v100.governance_contract,
-                    governance_percent: config_v100.governance_percent,
-                    astro_token_contract: config_v100.astro_token_contract,
-                    max_spread: config_v100.max_spread,
-                    rewards_enabled: false,
-                    pre_upgrade_blocks: 0,
-                    last_distribution_block: 0,
-                    remainder_reward: Uint128::zero(),
-                    pre_upgrade_astro_amount: Uint128::zero(),
-                };
-
-                CONFIG.save(deps.storage, &new_config)?;
-            }
-            _ => return Err(ContractError::MigrationError {}),
-        },
-        _ => return Err(ContractError::MigrationError {}),
-    }
-
-    set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
-
-    Ok(Response::new()
-        .add_attribute("previous_contract_name", &contract_version.contract)
-        .add_attribute("previous_contract_version", &contract_version.version)
-        .add_attribute("new_contract_name", CONTRACT_NAME)
-        .add_attribute("new_contract_version", CONTRACT_VERSION))
+pub fn migrate(_deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
+    Ok(Response::default())
```

Comment:
We need to return the migration error because we can forget to provide migration and data can be damaged by new logic